deployment/pypi/setup.py

```diff
@@ -50,6 +50,8 @@
     package_data = {'nni': ['**/requirements.txt']},
     python_requires = '>=3.5',
     install_requires = [
+        'azureml',
+        'azureml-sdk',
```

Comment:
azureml-sdk  is a very large package,  remove it from dependencies and let user to install it by themselves?

Reply:
removed from setup.py, add install information in document.